sessions/package-setup.qmd

```diff
@@ -8,18 +8,25 @@
 
 <!-- TODO: Add an introduction here -->
 
-**Session objectives**:
-
--   Identify when and why to create an R package.
--   Learn about and use the R packages `{devtools}` and `{usethis}` to
-    automate and streamline many package development tasks.
--   Learn how to create a new R package using
-    `usethis::create_package()`.
--   Describe and explain the purpose of the folders and files in a
-    skeleton R package.
--   Describe what are global defaults in R and how to use them to make
-    package development a bit easier.
--   Learn how and why you might add data to an R package.
+## Learning objectives
+
+The overall **learning outcome** for this session is:
+
+1.  Describe how an R package looks like and create a basic package
+    setup using tools that help streamline and assist with these initial
+    tasks.
+
+Specific **objectives** are:
+
+1.  Describe what a basic R package "physically" looks like, what the
+    required files and folders, and what their purpose is.
+2.  Use the `{usethis}` and `{devtools}` packages to create a basic R
```

Comment:
```suggestion
2.  Use the `{usethis}` and `{devtools}` packages to set up a basic R
```
I'm guessing this session is *setup*, while the overall course is *creating* a basic package